app/services/settings_addresser.rb

```diff
@@ -1,5 +1,22 @@
-class SettingsAddresser
-  def self.call(params)
-    Address.find_by(user_id: params[:user_id], type: params[:type]) || Address.new(user_id: params[:user_id], type: params[:type])
+class SettingsAddresser < Rectify::Command
+  def initialize(params)
+    @params = params
+  end
```

Comment:
Add variable `@address = find_or_initialize_by(user_id: @params[:user_id], type: @params[:type])` instead `find_address` and `new_address`
Also please use attr_reader for @params and @address